src/Apps/Artwork/artworkRoutes.tsx

```diff
@@ -62,8 +62,13 @@ export const artworkRoutes: AppRouteConfig[] = [
         }
       }
     `,
-    cacheConfig: {
-      force: true,
+    getCacheConfig: ({ context }) => {
```

Comment:
The ability to invoke this is new with the latest upgrade. We can do lots of things with this, and lots of route data comes back in the arguments